pkgs/games/dopewars/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, gtk3, autoconf, automake, ncurses, curl }:
+
+stdenv.mkDerivation {
+  pname = "dopewars";
+  version = "1.6.2";
+
+  src = fetchFromGitHub {
+    owner = "benmwebb";
+    repo = "dopewars";
+    rev = "86c3e8a841454440966693d571899fe4f738a4ae";
```

Use the version tag here, it is much cleaner than the commit hash
```suggestion
    rev = "v${finalAttrs.version}";
```